ui/app/components/app/confirmation-footer/confirmation-footer.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
Would this make more sense underneath the `confirmation` page directory? I doubt this will be used elsewhere.

Reply:
resolved in latest